diff --git a/dist/cli.js b/dist/cli.js
index 7871cebe46f6df886b76364db5346adfa1838622..cbb32659bb2a57c2cd847ddfd35a9a07ab22a033 100644
--- a/dist/cli.js
+++ b/dist/cli.js
@@ -110,7 +110,6 @@ async function run() {
 	};
 	let app = express();
 	app.disable("x-powered-by");
-	if (!isRSCBuild) app.use(compression());
 	let expressPublicPath = getExpressPath(build.publicPath);
 	app.use(path.posix.join(expressPublicPath, "assets"), express.static(path.join(build.assetsBuildDirectory, "assets"), {
 		immutable: true,
@@ -118,7 +117,6 @@ async function run() {
 	}));
 	app.use(expressPublicPath, express.static(build.assetsBuildDirectory));
 	app.use(express.static("public", { maxAge: "1h" }));
-	app.use(morgan("tiny"));
 	if (build.fetch) app.all("/{*splat}", createRequestListener(build.fetch));
 	else app.all("/{*splat}", createRequestHandler({
 		build: buildModule,
